modules/registry-configuring-storage-vsphere.adoc

```diff
@@ -87,7 +87,7 @@ storage:
     claim: <1>
 ----
 +
-<1> Leave the `claim` field blank to allow the automatic creation of an `image-registry-storage` PVC.
```

Comment:
swap/ will / is

We try to stay away from future tense/will. 

https://docs.engineering.redhat.com/display/CCSENABLE/Future+tense